testing/gmock/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
```

Comment:
Please use only tabs for indentation (and alignment) in APKBUILDs and other shell scripts, not spaces.